tests/test_requests.py

```diff
@@ -1466,6 +1466,18 @@ def test_environment_comes_after_session(self, httpbin):
         with pytest.raises(KeyError):
             proxies['http']
 
+    def test_merge_environment_settings_verify(self):
+        """Assert CA environment settings are merged as expected when missing"""
+        session = requests.Session()
+        assert session.trust_env is True
+        assert session.verify is True
+        assert 'REQUESTS_CA_BUNDLE' not in os.environ
+        assert 'CURL_CA_BUNDLE' not in os.environ
```

Comment:
This test needs to be adjusted slightly to patch the environment to make these things true. Otherwise this test will fail on machines where the user has set these environment variables on purpose.